### ISSUE TITLE
cyrillic input to WrapWithAbbr causes a dialog to stay open in TEA for Coda

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#116]
```
What steps will reproduce the problem?
1. Shift+Cmd+W (Wrap With Abbreviation)
2. Enter cyrillic 'з' instead of 'p' by mistake
3. Hit Enter key

What is the expected output? What do you see instead?
Instead of wrapping with <з> tag, nothing happened, but the dialog window
remained open. No reaction to Cancel, Ok, even after changing input.
Another call to WWA created a new dialog box instance which worked fine.
Yet the old one remained underneath.

What version of the product are you using? On what operating system?
Coda 1.6.10, MacOS X 10.5.8 Intel MacBookPro.

Please provide any additional information below.
Had to restart Coda.
```

---

Original issue reported on code.google.com by `serg...@gmail.com` on 25 Feb 2010 at 11:27

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `serge....@gmail.com` on 1 Feb 2011 at 10:56
- Added labels: **Milestone-0.7**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.7
```

---

Original comment by `serge....@gmail.com` on 2 Feb 2011 at 10:08
- Changed state: **Fixed**
